pkg/kubemark/hollow_proxy.go

```diff
@@ -42,6 +45,24 @@ type HollowProxy struct {
 	ProxyServer *proxyapp.ProxyServer
 }
 
+type FakeProxyHandler struct{}
```

Comment:
Removed FakeProxyHandler, but couldn't get rid of casting. Any suggestions how to do it?